src/Features/Core/Portable/ExternalAccess/UnitTesting/API/IUnitTestingSolutionCrawlerServiceAccessor.cs

```diff
@@ -1,14 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using Microsoft.CodeAnalysis.Host;
-
-namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
-{
-    internal interface IUnitTestingSolutionCrawlerServiceAccessor : IWorkspaceService
-    {
-        void Reanalyze(Workspace workspace, IUnitTestingIncrementalAnalyzerImplementation analyzer, IEnumerable<ProjectId> projectIds = null, IEnumerable<DocumentId> documentIds = null, bool highPriority = false);
```

Comment:
How will you access the APIs from unit testing layer? With restricted IVT being enforced, UnitTesting can only access APIs from ` Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api` namespace and we require these to be interfaces.

Reply:
@drognanar Looks like this interface is still in use in the below UnitTestingSolutionCrawlerServiceAccessor.cs file. Was it removed by accident?

Reply:
I renamed the other interface to be named the same as this one, so the interface ended up as an untracked file and I didn't commit it. Pushed new changes

Reply:
Thanks for noticing